lib/oauther.ex

```diff
@@ -94,7 +100,7 @@ defmodule OAuther do
   end
 
   defp nonce do
-    :crypto.rand_bytes(32) |> Base.encode64
+    :crypto.rand_bytes(16) |> Hexate.encode
```

Comment:
I was so sure I isolated the problem to nonce generation along the way, but obviously it was not the actual problem. Good catch.